order/src/main/java/com/caldevsupplychain/order/vo/OrderStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.caldevsupplychain.order.vo;
+
+public enum OrderStatus {
+	IN_PROCESS,
```

Comment:
I think we are going to have the following status.
`PENDING` -> `CONFIRMED` -> `IN_PROCESS` -> `DELIVERED` -> `FINISHED`